Assets/BossRoom/Scripts/Shared/ScriptableObjects/VisualizationConfiguration.cs

```diff
@@ -0,0 +1,72 @@
+using UnityEngine;
+
+namespace BossRoom
+{
+    /// <summary>
+    /// Describes how a specific character visualization should be animated.
+    /// </summary>
+    [CreateAssetMenu]
+    public class VisualizationConfiguration : ScriptableObject
+    {
+        [Header("Animation Triggers")]
+        [Tooltip("Trigger for when a player character is resurrected")]
+        [SerializeField] private string AliveStateTrigger = "StandUp";
+        [Tooltip("Trigger for when a player-character using this visualization becomes incapacitated")]
+        [SerializeField] private string FaintedStateTrigger = "FallDown";
+        [Tooltip("Trigger for when a monster using this visualization becomes dead")]
+        [SerializeField] private string DeadStateTrigger = "Dead";
+        [Tooltip("Trigger for when we expect to start moving very soon (to play a short animation in anticipation of moving soon)")]
+        [SerializeField] private string AnticipateMoveTrigger = "AnticipateMove";
+        [Tooltip("Trigger for when a new character joins the game and we are already a dead monster")]
+        [SerializeField] private string EntryDeathTrigger = "EntryDeath";
+        [Tooltip("Trigger for when a new character joins the game and we are already an incapacitated player")]
+        [SerializeField] private string EntryFaintedTrigger = "EntryFainted";
+
+        [Header("Other Animation Variables")]
+        [Tooltip("Variable that drives the character's movement animations")]
+        [SerializeField] private string SpeedVariable = "Speed";
+        [Tooltip("Tag that should be on the \"do nothing\" default nodes of each animator layer")]
+        [SerializeField] private string BaseNodeTag = "BaseNode";
```

Comment:
Just to apply Rider's code formatting suggestions here to reduce cleanup efforts. Explicitly this means removing the `private`s and if they're private, using `m_..`.

Reply:
ok